python/torch_mlir/extras/fx_importer.py

```diff
@@ -5,10 +5,16 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # Also available under a BSD-style license. See LICENSE.
 
+from sys import version_info
```

Comment:
Instead of doing a complicated version check, wdyt about:

```
try:
  from types import None type
except ImportError:
  NoneType = type(None)
```

Reply:
With a comment that less than 3.10 doesn't have it